eea/facetednavigation/upgrades/configure.zcml

```diff
@@ -5,6 +5,22 @@
   i18n_domain="eea">
 
   <genericsetup:upgradeSteps
+    source="7.5"
+    destination="8.4"
```

Comment:
In order for this step to show within Site Setup > Add-ons you'll have to also bump version within [profiles/default/metadata.xml](https://github.com/eea/eea.facetednavigation/blob/master/eea/facetednavigation/profiles/default/metadata.xml#L3)

Thank you.